google-cloud-firestore/lib/google/cloud/firestore/query.rb

```diff
@@ -1031,6 +1031,8 @@ def snapshot_to_cursor snapshot, query
           values_to_cursor values, query
         end
 
+        # rubocop:disable Performance/ReverseEach
+
```

Comment:
Again, could we change the code to call `inequality_paths.reverse_each` instead of disabling this cop?